test/integration/029_docs_generate_tests/test_docs_generate.py

```diff
@@ -1302,7 +1300,6 @@ def expected_postgres_references_manifest(self, model_database=None):
                     'original_file_path': docs_path,
                     'package_name': 'test',
                     'path': 'docs.md',
-                    'resource_type': 'docs',
```

Comment:
Since docs are under their own key in the manifest, this seemed ok to me...

Reply:
I had a comment (can't remember if it was my own notes or from the actual code) about removing it because docs already are weird, and how we only needed it for logging of rendering errors. It was easy to do and I am tired of it being another parameter to go wrong when I'm building objects by hand in IPython... tiny use case but it wasn't really any effort.